src/API/Google/AdsCampaign.php

```diff
@@ -176,21 +185,42 @@ public function get_campaign( int $id ): array {
 	 */
 	public function create_campaign( array $params ): array {
 		try {
+			$base_country = $this->container->get( WC::class )->get_base_country();
+
+			$location_ids = array_map(
+				function ( $country_code ) {
+					return $this->google_helper->find_country_id_by_code( $country_code );
+				},
+				$params['targeted_locations']
+			);
+
+			$location_ids = array_filter(
+				$location_ids,
+				function ( $location_id ) {
+					return isset( $location_id );
+				}
+			);
```

Comment:
For `array_filter` it mentions:
> If no callback is supplied, all empty entries of array will be removed.

```suggestion
			$location_ids = array_filter( $location_ids );
```
We'd be filtering out more than just null but that shouldn't be harmful.

Reply:
Nice, updated in be14290, thanks!